core/app/models/spree/order.rb

```diff
@@ -404,7 +404,7 @@ def available_payment_methods
     end
 
     def insufficient_stock_lines
-      line_items.select(&:insufficient_stock?)
+      line_items.includes(variant: :product).select(&:insufficient_stock?)
```

Comment:
@nimish-mehta I think we need to move this at the controller level. Or check whether it's already included or not.